README.md

```diff
@@ -1,8 +1,8 @@
 # DER-scheduling
-IEC-scheduling (or DER-scheduling) is a specific form of scheduling with a lot of properties that should not be confused with any [generic scheduling](https://en.wikipedia.org/wiki/Schedule) such also as in IT.
+IEC-scheduling (or DER-scheduling) is a specific form of scheduling with a lot of properties that should not be confused with any [generic scheduling](https://en.wikipedia.org/wiki/Schedule) such also as in IT. 
 
 ## Introduction
-This project aims to create a production grade open-source scheduling software stack, enabling consumers, manufacturers and automations suppliers to implement IEC Scheduling according to IEC 61850-7-4-2010 Annex K. Formaly known as TR IEC 61850-90-10. The main use-case is for controling Distributed Energy Resources (DER), but not limited to that. 
+This project aims to create a production grade open-source scheduling software stack, enabling consumers, manufacturers and automations suppliers to implement IEC Scheduling according to IEC 61850-7-4-2010 Annex K. Formaly known as TR IEC 61850-90-10. The main use-case is for controling Distributed Energy Resources (DER), but not limited to that. The software is (being) integrated to become useful for interacting with communication protocols in [project ReLevENT](https://github.com/alliander-opensource/ReLevENT). See also here for [use-cases](https://github.com/alliander-opensource/ReLevENT/blob/main/usecases/use-cases-edge-view.md) where it can be applied to.
```

Comment:
Why is the benefit of starting a new repository? RelevENT? I don't get the differences.

Reply:
The idea is that the ReLevENT repository contains only the additional code for the fledge plugin that is based on the DER-Scheduler code. The DER Scheduler code that is independent from Fledge will stay in its own repository.

Reply:
@Sander3003 Yes, this is what we agreed on to have the DER-scheduling remaining the core and ReLevENT becomes an integration project.